README.md

```diff
@@ -1,2 +1,20 @@
 # deep_sentiment140
-Deep learning approaches for sentiment140 dataset.
+
+Deep learning approaches for Sentiment140 dataset.
+
+## Prerequisites
+
+You need to have Python 3.6 (at least) installed on your system. To run this project, commands are the following.
```

Comment:
```suggestion
 [Python](https://www.python.org/) 3.6 or higher is required.
 
 All python requirements can be found in file `requirements.txt` and can be installed using
 
 ```shell
 pip install -r requirements.txt
 ```
```

Reply:
Corrected in https://github.com/XanX3601/deep_sentiment140/pull/3/commits/4833fdfb78d9b6d5d7314cc5c237879d8ad4c360.